packages/contracts-bedrock/src/universal/interfaces/IOwnable.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/// @title IOwnable
+/// @notice Interface for Ownable.
+interface IOwnable {
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    function owner() external view returns (address);
+    function renounceOwnership() external;
+    function transferOwnership(address _newOwner) external;
```

Comment:
I don't know if semgrep will allow this (or if we'll all in agreement from a style perspective), but for interfaces it would be nice if we can drop the underscore prefix/postfix convention. This way when users or us use the interface with named arguments it's a little cleaner.

Reply:
Agree, semgrep does complain for now though.

Reply:
When using named args like `myContract.foo({bar: 1, baz: 2})`, changing the arg names is a breaking change for interface consumers, so if we do want to remove the underscores we should do it earlier rather than later. Will defer to you on deciding if/when to handle that

Reply:
Mmmm yeah good point. I guess we can just put it in even if semgrep complains. Maybe we just add semgrep ignores for these files?

Reply:
Yea should we just add any `interfaces/` folder to the `.semgrepignore` file? We probably should confirm there are no semgrep rules we do want to run for the interfaces first, just in case

Reply:
For now I'll do a manual semgrep ignore and we can figure this out later